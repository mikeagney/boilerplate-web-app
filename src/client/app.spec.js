import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './app';

describe('App root container', () => {
  it('will render the root app', () => {
    // Arrange
    const wrapper = shallow(<App />);

    // Act
    // Assert
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
