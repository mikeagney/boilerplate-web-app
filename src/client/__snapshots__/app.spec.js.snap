// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App root container will render the root app 1`] = `
<h1>
  Hello World from React!
</h1>
`;
